Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM ruby:2.2.4
+# docker 17.06+ will let us do this:
```

Comment:
Nice! Can't wait for that patch release :)

We could use that new feature in several places.